tools/ci_build/github/azure-pipelines/linux-ort-srv-ci-pipeline.yml

```diff
@@ -1,19 +1,79 @@
 jobs:
 - job: Debug_Build
-  pool: Hosted Ubuntu 1604
+  pool: Linux-CPU
```

Comment:
Any reason to switch from Hosted Ubuntu 1604 agents to self hosted ones? Are the hosted one taking much longer?

Reply:
The `Hosted Ubuntu 1604` could never finish the full steps in 1 hour. It also does not contains the test data set.